1_fetch/src/get_data.R

```diff
@@ -0,0 +1,7 @@
+# Get the data from ScienceBase
+get_data_from_sb <- function(sbid, outer_dir){
```

Comment:
Nice, I like that you're parameterizing the output directory rather than letting it stay hardcoded!